front-end/src/util/sorting.test.js

```diff
@@ -187,4 +187,30 @@ describe("Sort by last met date", () => {
     expect(craftspeople[2].id).toBe(3);
     expect(craftspeople[3].id).toBe(2);
   });
+
+  it("ARNAUD DATA", () => {
```

Comment:
the name should be something like `should order craftspeople by last meeting` instead of `ARNAUD DATA`

Reply:
test name changed